libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -84,7 +84,8 @@ export class SdsFiltersComponent implements OnInit, OnChanges {
   _isEmpty = (obj: any): boolean => Object.keys(obj).length === 0;
   overwrite = (baseObj: any, newObj: any) => {
     const result = {};
-    for (const key in baseObj) {
+    const mergedObj = {...baseObj, ...newObj};
+    for (const key in mergedObj) {
```

Comment:
Use a merged obj because previous logic skips over any keys that exist in new object but not in baseObj